src/trinary_sse42.h

```diff
@@ -9,8 +9,13 @@
 #ifndef TRINARY_SSE42_H_
 #define TRINARY_SSE42_H_
 
+#if defined(__SSE4_2__)
 #include <nmmintrin.h>
+#endif
 #include "constants.h"
+#if defined(__ARM_NEON)
+#include "sse2neon.h"
+#endif
```

Comment:
Add `#error` when both SSE4.2 and Arm NEON are not available.

Reply:
Done.